### PR TITLE
alt text updated

### PR DESCRIPTION
**Pull request by jhhornn**
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/games/tutorials/2d_breakout_game_pure_javascript/move_the_ball/index.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ function draw() {
 
 Save your code again and try it in your browser. This works OK, although it appears that the ball is leaving a trail behind it:
 
-![](ball-trail.png)
+![a white background with a thick blue line drawn at 45 degrees from the right and not touching the base](ball-trail.png)
```
Comment (estelle) [inline files/en-us/games/tutorials/2d_breakout_game_pure_javascript/move_the_ball/index.md:93]:
```suggestion
![A thick blue line going up and to the right at 45 degrees](ball-trail.png)
```

**Review by estelle — APPROVED** [review]
thanks

**Comment by estelle** [comment]
oops, someone else already did this one. Conflicting file. So closing.

**Comment by jhhornn** [comment]
Ohhh

It wasn't indicated on the spreadsheet

**Comment by jhhornn** [comment]
Also, did I do this without including the issue number?

I just want to make sure that I followed your previous instruction.

**Pull request closed without merging**